### DISCUSSION TITLE
Unable to renew Github Pro for Student

### DISCUSSION THREAD

**Discussion by merweeeeen** [community/community#140894] [GitHub Education]
### Hi everyone,

Hi Everyone,

I recently received an email from GitHub stating that my GitHub Pro for Students subscription is expiring soon, and I’ve been trying to renew it. However, I’m currently on an exchange program and don’t have access to my local student ID. My student ID also doesn’t show a validity date, which I believe caused my previous renewal attempt to be rejected. 

The current application process asks for a picture of my student ID, transcript, or class schedule, but unfortunately, I don’t have access to any of these documents at the moment, I tried using a screenshot of the unofficial transcript I have through my phone and it didn't work. What would be the best way to proceed in this situation?

Thanks in advance !

**Comment by steeleb** [comment]
I am also having this issue - the language seems to imply that you can 'upload' documents, but I've only been able to figure out how to use my laptop/cellphone camera. Is there a way to upload this documentation instead of take a picture?
